app/assets/javascripts/application.js

```diff
@@ -556,3 +578,9 @@ function disableButtonToggle(item, explicit) {
 
   $(item).blur();
 }
+
+function icon_text(icon_class, name, inner_text) {
+  var icon = '<span class="' + icon_class + "-" + name + '"/>'
```

Comment:
Why not add the icon_class twice instead of passing "pficon pficon" all the time? 
Also please try to make the js function and the rb function that do this more similar, the order is reversed between them making it harder to remember which parameter is which.